config.json

```diff
@@ -34,6 +34,10 @@
         "site": "us-east-2",
         "type": "aws_s3"
     }],
+    "backbeat": {
```

Comment:
To make it usable out of the box and ease testing, could we handle `CRR_METRICS_HOST` and `CRR_METRICS_PORT` environment variables to update this directly in `docker-endpoint.sh`?

Reply:
You got it! Updated so that you can start the image with a custom endpoint as such:

```
docker run -e CRR_METRICS_HOST=10.100.3.139 -e CRR_METRICS_PORT=1337 a7ae6fa66070
```